### PR TITLE
Use date formatting library

### PR DESCRIPTION
**Pull request by neutmute**
* Make available and use date formatting library (#22)
* Update docs

### CODE REVIEW

**Comment by neutmute** [comment]
Any chance you can take a look at this @piuccio? I have a follow up one in the works to add a few more enhancements :) 
Merging this first will reduce the diff

File: package.json [hunk 1]
```diff
@@ -12,21 +12,25 @@
...
     "debug": "^2.6.0",
     "ejs": "^2.5.5",
     "optimist": "^0.6.1"
   },
-  "contributors": {
-    "name": "Fabio Crisci",
-    "email": "fabio@ariatemplates.com",
-    "url": "https://github.com/piuccio"
-  },
+  "contributors": [
```
Comment (piuccio) [inline package.json]: since you've modified this, you can add your name if you like

**Comment by piuccio** [comment]
looks much cleaner thanks, I'll merge it now, feel free to add you name in the contributors in the follow up PR

**Comment by neutmute** [comment]
awesome - thanks.
I won't worry about the `contributors` - I just cleaned it up as Visual Studio Code was complaining that it wasn't correct and I wanted to clear the warning :)

**Pull request merged**